### DISCUSSION TITLE
Marquer les coups illégaux

### DISCUSSION THREAD

**Discussion by wilchel** [pascalaubry/papi-web#25] [Ideas]
Bonjour
Merci pour cet outil  PapiWeb :)
Une proposition sur papi web , que les arbitres puissent mettre des annotations de coups illégaux  sur les parties.
Sur une compétition de très jeunes, il peut y avoir plusieurs arbitres et si un coup illégal se passe , un autre arbitre pourra identifier si c est le premier ou le deuxième.

**Comment by Amaras** [comment]
Bonjour @wilchel,
C'est une très bonne idée, que nous ne pouvons malheureusement pas implémenter facilement actuellement, notre format de stockage étant trop proche du format utilisé par papi.

Cependant, ce format de stockage ne répond déjà pas correctement à nos attentes, donc nous prévoyons de le changer dans une prochaine version.

La fonctionnalité que vous proposez est, je pense, relativement simple à ajouter, nous pouvons l'ajouter à la liste de chose à faire pour la prochaine version.

**Comment by Amaras** [comment]
Fonctionnalité ajoutée par la version 2.2, je ferme cette discussion, et un grand merci pour l'idée !
